### PR TITLE
III-3757 fix php stan warnings

### PR DESCRIPTION
**Pull request by grubolsch**
### Changed
Removed phpstan ignore comments where possible.
When not possible I added comments if it could be fixed when upgrading to PHP 8

---
Ticket: https://jira.uitdatabank.be/browse/III-3757

### CODE REVIEW

File: tests/CalendarTest.php [hunk 1]
```diff
@@ -107,39 +107,6 @@ public function setUp(): void
         );
     }
 
-    /**
-     * @test
-     */
-    public function time_stamps_need_to_have_type_time_stamp(): void
```
Comment (LucWollants) [inline tests/CalendarTest.php:113]: Why are these tests removed?

Comment (LucWollants) [inline tests/CalendarTest.php:113]: I would keep the tests, what if PHPStan would be removed from the project?

Comment (grubolsch) [reply tests/CalendarTest.php:113]: restored tests

**Comment by grubolsch** [comment]
This pr is ready for review. All feedback was processed.

**Review by LucWollants — APPROVED** [review]

**Pull request merged**